doc/mgr/dashboard.rst

```diff
@@ -7,98 +7,98 @@ Overview
+out as a simple read-only view into run-time information and performance
+data of Ceph clusters. It used a very simple architecture to achieve the
+original goal. However, there was growing demand for richer web-based
 management capabilities, to make it easier to administer Ceph for users that
-prefer a WebUI over using the command line.
+prefer a WebUI over the CLI.
 
-The new :term:`Ceph Dashboard` module is a replacement of the previous one and
-adds a built-in web based monitoring and administration application to the Ceph
-Manager. The architecture and functionality of this new module is derived from
+The new :term:`Ceph Dashboard` module adds web based monitoring and
+administration to the Ceph Manager. The architecture and functionality of this new
+module are derived from
 and inspired by the `openATTIC Ceph management and monitoring tool
-<https://openattic.org/>`_. The development is actively driven by the team
-behind openATTIC at `SUSE <https://www.suse.com/>`_, with a lot of support from
-companies like `Red Hat <https://redhat.com/>`_ and other members of the Ceph
+<https://openattic.org/>`_. Development is actively driven by the
+openATTIC team at `SUSE <https://www.suse.com/>`_, with support from
+companies including `Red Hat <https://redhat.com/>`_ and members of the Ceph
 community.
 
-The dashboard module's backend code uses the CherryPy framework and a custom
-REST API implementation. The WebUI implementation is based on
-Angular/TypeScript, merging both functionality from the original dashboard as
-well as adding new functionality originally developed for the standalone version
-of openATTIC. The Ceph Dashboard module is implemented as a web
-application that visualizes information and statistics about the Ceph cluster
-using a web server hosted by ``ceph-mgr``.
+The dashboard module's backend code uses the CherryPy framework and implements
+a custom REST API. The WebUI implementation is based on
+Angular/TypeScript and includes both functionality from the original dashboard
+and new features originally developed for the standalone version
+of openATTIC. The Ceph Dashboard module is implemented as an
+application that provides a graphical representation of information and statistics
+through a web server hosted by ``ceph-mgr``.
 
 Feature Overview
 ^^^^^^^^^^^^^^^^
 
 The dashboard provides the following features:
 
 * **Multi-User and Role Management**: The dashboard supports multiple user
-  accounts with different permissions (roles). The user accounts and roles
-  can be modified on both the command line and via the WebUI. The dashboard
-  supports various methods to enhance password security, e.g. by enforcing
-  configurable password complexity rules, forcing users to change their password
+  accounts with different permissions (roles). User accounts and roles
+  can be managed via both the command line and the WebUI. The dashboard
+  supports various methods to enhance password security.  Password
```

Comment:
```suggestion
  supports various methods to enhance password security. Password

Reply:
Do we have a standard for spaces after periods?  I would expect modern SW to render one and two the same.  I'm pushing this change nonetheless.